tests/conftest.py

```diff
@@ -48,7 +48,7 @@ def translate_path(self, path):
         class ThreadingHTTPServer(ThreadingMixIn, HTTPServer):
             daemon_threads = True
 
-    with ThreadingHTTPServer(('', 0), Handler) as server:
+    with ThreadingHTTPServer(('localhost', 0), Handler) as server:
```

Comment:
Ah man!!

Reply:
hey no problem. 
yep that was it, Windows didn't like the 0.0.0.0 address :)